### DISCUSSION TITLE
Github skyline hangs

### DISCUSSION THREAD

**Discussion by clivet268** [community/community#40856] [Other Feature Feedback, Questions, & Ideas]
This is about the 3d visualizer of your github activity, an example is [https://www.githubskyline.com/clivet268/2022](url) but none of them work for me. I set it as my homepage for a nice background in browser but its possible I was accessing it too much and got IP banned? If not that I have no idea why it just stopped one day. The time it stopped working was around when I did a partial reset of windows but Im not sure how that would affect accessing skyline

Any insight would be helpful

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @clivet268! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
